pages.fr/common/brew-bundle.md

```diff
@@ -0,0 +1,28 @@
+# brew bundle
+
+> Outil de gestion de paquets pour Homebrew, Homebrew Cask et le Mac App Store.
```

Comment:
```suggestion
> Bundler pour Homebrew, Homebrew Cask et le Mac App Store.
```

Reply:
I'm not quite sure "Bundler" will be easily understood by french speaking users...
Maybe a second opinion can help us on this ?